helpers/scss.js

```diff
@@ -1,13 +1,14 @@
-module.exports = function(gulp, plugins, config, name, locale, file) {
+'use strict';
+module.exports = function (gulp, plugins, config, name, locale, file) {
   // Return function that is executed inside of .pipe()
   return () => {
-    var theme      = config.themes[name],
-        src        = file || config.projectPath + theme.src + '/**/*.scss',
-        dest       = config.projectPath + theme.dest + '/' + locale,
-        maps       = plugins.util.env.maps || false,
-        production = plugins.util.env.prod || false,
-        postcss    = [],
-        parentPath = require('./parent-theme-dir')(name, config);
+    const theme      = config.themes[name],
+          src        = file || config.projectPath + theme.src + '/**/*.scss',
+          dest       = config.projectPath + theme.dest + '/' + locale,
+          maps       = plugins.util.env.maps || false,
+          production = plugins.util.env.prod || false,
+          postcss    = [],
```

Comment:
`const` prevents a variable from being reassign like so `postcss = ['example']` `postcss.push('example')` is perfectly fine 👍 same goes for objects. Assigning keys is allowed. Just not reassigning. the variable. That's why you can use const almost everywhere in the codebase. Just a few exceptions. Mathias Bynens has a great article about it: https://mathiasbynens.be/notes/es6-const